CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -69,15 +74,15 @@ def _get_user_agent_header():
     return headers
 
 
-# SQL Admin
-def build_sql_admin_service(credentials=None):
-    """Authorizes and sets custom CloudSQL Replicator user agent.
-
-      Args:
-        credentials (google.auth.Credentials): Credentials to authorize client
+def build_authorized_svc(service, version, credentials=None):
+    """Builds an authorized service proxy with customer user agent.
```

Comment:
Did you mean custom insead of customer UA?

Reply:
Yes. Fixed.